Assets/Mirage/Runtime/Transport/UdpSocket/UdpTransport.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Sockets;
+using Mirage.SocketLayer;
+using UnityEngine;
+
+namespace Mirage
+{
+    public sealed class UdpTransport : SocketCreator
+    {
+        [SerializeField] int port;
+
+        public override ISocket CreateClientSocket()
+        {
+            return new UDPSocket();
+        }
+
+        public override ISocket CreateServerSocket()
+        {
+            return new UDPSocket();
+        }
+
+        public override EndPoint GetBindEndPoint()
+        {
+            return new IPEndPoint(IPAddress.Any, port);
+        }
+        public override EndPoint GetConnectEndPoint(string address)
+        {
+            return new IPEndPoint(IPAddress.Parse(address), port);
+        }
+
+        public override bool ClientSupported => platformNotWebgl;
+        public override bool ServerSupported => platformNotWebgl;
+        private static bool platformNotWebgl => Application.platform != RuntimePlatform.WebGLPlayer;
+    }
+
+    public class UDPSocket : ISocket
+    {
+        readonly Socket socket;
+
+        public UDPSocket()
+        {
+            socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
```

Comment:
Fholm uses some additional configuration for the socket to work properly on Windows. Should probably check his repo.

Reply:
Blocking = false seems to be the only extra thing, was there something else as well?

Reply:
Yeah, there was another config for something with a shortcut as its name but I can't remember what it was. Should be on his stream socket repo.